### ISSUE TITLE
Add a way for moderators to send custom message via notifications

### ISSUE THREAD

**Issue by jezznar** [vck3000/ProAvalon#29]
This would be a good way to send messages to everybody and notify them about something worth notable.
Here's a simple wireframe of the modal:

Message Editor
![image](https://user-images.githubusercontent.com/13676977/45255877-a2218a80-b3c0-11e8-9c85-ea69997b2495.png)

Message modal when notification is clicked
![image](https://user-images.githubusercontent.com/13676977/45255886-d85f0a00-b3c0-11e8-8c4e-8d916c74b62a.png)

**Comment by vck3000** [comment]
Cool, sounds like a good idea.

Do you want it to send to all players who are currently online? Or store it as a new popup notification for everyone when they log in?

**Comment by jezznar** [comment]
Wait, doesn't the current notification system store the notification and also updates all online players?

**Comment by bocaben** [comment]
why not both? btw, an interface is nice but not needed - CLI is fine:
/mservernotifyall <msg> - Have the server leave a notification to all users.
/mservermessageall <msg> - Have the server message all online users.

also, perhaps it should officially come from "The Server"/"The Mod Team" with maybe an "(Issued by <Mod's Name>)" after it.

not sure what u had in mind for this feature, but could be nice to have in case we need a way to notify everyone about something, i guess, if so, this is how i would do it.

**Comment by jezznar** [comment]
>why not both? btw, an interface is nice but not needed

You wouldnt like popup as a player while playing, would you? Besides it's a UX thing to not obstruct players' views too much while gaming.

>CLI is fine:
>/mservernotifyall - Have the server leave a notification to all users.
>/mservermessageall - Have the server message all online users.

I like this

**Comment by bocaben** [comment]
Was talking about mod-side, not user-side. For /mservermessageall I think the user should see it in both all chat and main chat like /mwhisper. No pop-ups, although this feature will be used sparingly, so it's not like it'd be a constant bother

**Comment by vck3000** [comment]
Fixed in 6f1bb1ae27e8d658ae5328f1d20593e8682dbbae.

Notifications are done through socket.js's:
```
const newUpdateNotificationRequired = 1551095116448;
const updateMessage = `message`;
```
